Formula/pyqt5.rb

```diff
@@ -15,8 +16,8 @@ class Pyqt5 < Formula
 
   deprecated_option "enable-debug" => "with-debug"
 
-  depends_on :python3 => :recommended
-  depends_on :python => :optional
+  depends_on :python => :recommended
+  depends_on :python3 => :optional
```

Comment:
I dropped my changeset to pyqt5. However, things are still inconsistent among pyqt,pyqt5 and qscintilla2.

Therefore I still ask in `qscintilla2.rb` for explicit `pyqt --with-python3` and `pyqt5 --with-python3` (albeit the latter is default, the former not).